src/test/java/org/biouno/unochoice/jenkins_cert_1954/TestParametersXssVulnerabilities.java

```diff
@@ -92,7 +93,18 @@ public void testChoicesParameterXss() throws IOException, SAXException {
                 }
             }
             assertNotNull("Could not locate rendered parameter element", renderedParameterElement);
-            String renderedText = renderedParameterElement.getFirstChild().asXml();
+
+            DomNode firstChild = null;
+            for (DomNode child : renderedParameterElement.getChildren()) {
+                // Spinner element must be ignored
+                firstChild = child;
+                if (child instanceof HtmlDivision && ((HtmlDivision) child).getAttribute("id").endsWith("-spinner")) {
+                    continue;
+                }
+                break;
+            }
+            assertNotNull("Could not locate first child element", firstChild);
+            String renderedText = firstChild.asXml();
```

Comment:
Thought it would be quite slow, but I think ~11 seconds on my old laptop should be fine for this acceptance test. We can improve it later if necessary. A good workaround :+1: 

```bash
[INFO] Running org.biouno.unochoice.jenkins_cert_1954.TestParametersXssVulnerabilities
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 11.86 s -- in org.biouno.unochoice.jenkins_cert_1954.TestParametersXssVulnerabilities
```